warnings.md

```diff
@@ -172,3 +172,10 @@ See jQuery-ui [commit](https://github.com/jquery/jquery-ui/commit/c0093b599fcd58
 **Cause:** Calling `.toggleClass()` with no arguments, or with a single Boolean `true` or `false` argument, has been deprecated. Its behavior was poorly documented, but essentially the method saved away the current `class` value in a data item when the class was removed and restored the saved value when it was toggled back. If you do not believe you are specificially trying to use this form of the method, it is possible you are accidentally doing so via an inadvertent undefined value, as `.toggleClass( undefined )` toggles all classes.
 
 **Solution:** If this functionality is still needed, save the current full `.attr( "class" )` value in a data item and restore it when required.
+
+### JQMIGRATE: jQuery.event.props are deprecated and removed
+### JQMIGRATE: jQuery.event.fixHooks are deprecated and removed
```

Comment:
We never documented them, although we did [document these props and fixHooks](https://learn.jquery.com/events/event-extensions/). It would be possible to add them but I'd prefer to wait until we get some sign that people were modifying them since it will add some more overhead for every event to see if anything was added.